packages/braid-design-system/src/lib/components/private/Modal/Modal.css.ts

```diff
@@ -12,9 +13,16 @@ export const fixedStackingContext = atoms({
 
 export const resetStackingContext = atoms({ position: 'relative', zIndex: 0 });
 
-export const backdrop = style({
-  background: 'rgba(0, 0, 0, .4)',
-});
+export const backdrop = style(
+  colorModeStyle({
+    lightMode: {
+      background: 'rgba(0, 0, 0, .4)',
+    },
+    darkMode: {
+      background: 'rgba(0, 0, 0, .6)',
```

Comment:
Should we be using hex codes (polished `rgba`) for better browser support?

Reply:
Hmm, possibly. Docs seems a little vague, but maybe missing support Safari 9-12. If that is the case, might be worth sweeping the code base as a follow up, as i can see a few around.

Reply:
Just confirming for this instance though, we cant use polished as we need transparency. Given its the backdrop as a standalone element, i'll split it into `background` and `opacity` properties.